### PR TITLE
Add package sitmo

### PR DESCRIPTION
**Pull request by jxy**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -6288,5 +6288,19 @@
...
     "license": "MIT",
     "web": "https://github.com/jangko/freetype"
+  },
+  {
+    "name": "sitmo",
+    "url": "https://github.com/jxy/sitmo",
+    "method": "git",
+    "tags": [
+      "RNG",
+      "Sitmo",
```
Comment (dom96) [inline packages.json]: For the future: this tag is redundant.

**Comment by dom96** [comment]
thanks!

File: packages.json (hunk 1 above)
Comment (jxy) [inline packages.json]: I see.  Thanks.

Comment (dom96) [inline packages.json]: No problem, Nimble should enforce that really :)

**Pull request merged**